src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/CoverageModelEMWorkspaceMathUtils.java

```diff
@@ -0,0 +1,117 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
+
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.math3.linear.*;
+import org.apache.commons.math3.util.Precision;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.tools.coveragemodel.nd4jutils.Nd4jApacheAdapterUtils;
+import org.nd4j.linalg.api.ndarray.INDArray;
+import org.nd4j.linalg.factory.Nd4j;
+import org.nd4j.linalg.ops.transforms.Transforms;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Common math utilities for {@link CoverageModelEMWorkspace}
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class CoverageModelEMWorkspaceMathUtils {
+
+    /**
+     * Inverts a square INDArray matrix using Apache LU decomposition
+     *
+     * @param mat matrix to be inverted
+     * @return inverted matrix as an {@link INDArray}
+     */
+    public static INDArray minv(@Nonnull final INDArray mat) {
```

Comment:
`minv`, `eig`, `linsolve`, etc are standard Fortran-era math functions that people are used to. I'd rather keep them "standard" and not java-verbose.